chemprop/cli/predict.py

```diff
@@ -162,21 +163,18 @@ def process_predict_args(args: Namespace) -> Namespace:
     return args
 
 
-def main(args):
-    match (args.smiles_columns, args.reaction_columns):
-        case [None, None]:
-            n_components = 1
-        case [_, None]:
-            n_components = len(args.smiles_columns)
-        case [None, _]:
-            n_components = len(args.reaction_columns)
-        case _:
-            n_components = len(args.smiles_columns) + len(args.reaction_columns)
+def find_models(model_path: Path):
+    if model_path.suffix in [".ckpt", ".pt"]:
+        return [model_path]
+    elif model_path.is_dir():
+        return list(model_path.rglob("*.ckpt")) + list(model_path.rglob("*.pt"))
 
-    multicomponent = n_components > 1
 
+def make_prediction_for_model(
+    args: Namespace, model_path: Path, multicomponent: bool, output_dir: Path
```

Comment:
`output_dir` isn't used. Instead `args.output` is written to.

Reply:
Thanks for catching this!